### DISCUSSION TITLE
Next 13 /app features stability

### DISCUSSION THREAD

**Discussion by ErAz7** [vercel/next.js#42487] [Help] [answered]
Hello there
I've just recently noticed next 13's app folder and many cool features it offres by implementing RSC and Suspense,
Now the point is, this feature is in Beta phase so obviously it means unexpected future (potentialy breaking) changes, But the question here is, how breaking these changes are potential to be. To be more clear, I'm of course okay to take the risk of using this Beta feature in my production app and future changes are okay but of course only as long as these changes are not fundamental ones, like total changes in the file structures, data fetching in components, etc

Is there any overview on this in vercel? or better say, is it possible to have such an overview?

**Comment by leerob** [comment, marked as answer]
You can follow along on the roadmap here: https://beta.nextjs.org/docs/app-directory-roadmap

In general, we don't recommend deploying `app` directory to production yet, as it's in beta. As we move further in stability, we will update our guidance. Thank you!

**Reply by ErAz7** [reply]
Thanks @leerob, this roadmap is all I need

**Reply by ayhanap** [reply]
I don’t see a timeline on the roadmap, will /app be stable in an upcoming minor version of next.js ?

**Reply by ErAz7** [reply]
well after checking the roadmap, agree with @ayhanap

**Reply by leerob** [reply]
No timeline defined - we will share updates as we make progress 😄 On the scale of months, not years. Software estimation is hard!

**Reply by ayhanap** [reply]
I can understand that these features heavily depend on core react features. Will help by using and giving feedback about it ✌️

**Reply by ErAz7** [reply]
months is somewhat quick enough, at leaset for my use case, thanks to the Next team
